app/BUILD.bazel

```diff
@@ -591,6 +592,7 @@ android_library(
         artifact("androidx.constraintlayout:constraintlayout:1.1.3"),
         artifact("androidx.lifecycle:lifecycle-livedata-core:jar:2.0.0-alpha1"),
         artifact("androidx.lifecycle:lifecycle-livedata-ktx"),
+        artifact("com.chaos.view.PinView"),
```

Comment:
This should match the other artifact: ``com.chaos.view:pinview:1.4.4``.

Reply:
Made the change. I hope the tests run correctly this time.